CMakeLists.txt

```diff
@@ -68,3 +75,6 @@ endif ()
 #    DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/bin/hello_2
 #    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
 #)
+
+
+add_subdirectory(test)
```

Comment:
This line is not neeeded, look at comments on your previous pull request

Reply:
Commenting it is giving error.

Reply:
Please read my comments carefully, we already have add_subdirectory(test), you just need to turn on the flag BUILD_TESTS , similar to what you did to VCPKG in Clion

Reply:
```
#Setup test project
option(BUILD_TESTS "Build all tests." ON) # Makes boolean 'test' available.
if (BUILD_TESTS)
#  find_package(Catch2 CONFIG REQUIRED)
  add_subdirectory(${EXT_PROJECTS_DIR}/catch)

  set(PROJECT_TEST_NAME masim_test)

add_subdirectory(test)
endif ()
```

Like This?

Reply:
Yes, but you dont have to do anything here, it's already there

Reply:
Do I need to make changes in if() its configured for Catch?

Reply:
Sure, you don't need this line also.
add_subdirectory(${EXT_PROJECTS_DIR}/catch)

The main idea here is you don't have to add_subdirectory(test) twice

Reply:
Okay. Thanks.